postcss.config.js

```diff
@@ -34,7 +34,11 @@ module.exports = ({ file, options, env }) => {
   return {
     plugins: [
       require("postcss-bidirection"),
-      require("autoprefixer"),
+      require("autoprefixer")({
+				browsers: ['> 3%', 'not ie <= 10'],
```

Comment:
Haha yeah. Let’s do latest chrome and latest Firefox

Reply:
should I change this to `['last 2 Firefox versions', 'last 2 Chrome versions']` ?

note: browser list is there: https://github.com/ai/browserslist#queries

Reply:
sure

Reply:
done